InfoLogger/test/lib/mocha-sqldatasource.js

```diff
@@ -0,0 +1,221 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const SQLDataSource = require('../../lib/SQLDataSource.js');
+const {MySQL} = require('@aliceo2/web-ui');
+
+
+describe('SQLDataSource', () => {
+  const config = {
+    mysql: {
+      host: 'localhost',
+      user: 'root',
+      password: 'admintest',
```

Comment:
We can't have any password in the code, even dummy ones.

Reply:
them